vendor/github.com/robert-milan/go-object-interning/object_intern_config.go

```diff
@@ -0,0 +1,32 @@
+package goi
+
+// Types of compression
+const (
+	NOCPRSN = iota
```

Comment:
i recommend you type alias uint8 to `Compression`, then this value can simply be `None`. the fact that it is about compression is clear from the type. + we get to benefit from static type checks.
note also that in Go we use camelcase even for constants.